test/python/model_validation/validation_tool.py

```diff
@@ -0,0 +1,184 @@
+import onnxruntime_genai as og
```

Comment:
you can move the tool to the https://github.com/microsoft/onnxruntime-genai/tree/main/tools/python